_includes/partners/x-col.html

```diff
@@ -6,7 +6,7 @@
         <div class="item tablet:grid-col">
           {% if item.image and item.image != '' %}
             <img
-              alt=""
+              alt="{% if item.alt and item.alt != ''%}{{ item.alt }}{% endif %}"
```

Comment:
Can we make this non-optional? Or at the very least, force the item to include an explicitly-empty value, so that we can turn it from an opt-in to and opt-out? Alternative text should rarely be empty.

```suggestion
              alt="{% if item.alt != ''%}{{ item.alt }}{% endif %}"
```

I expect this will cause some issues because the logos don't have alt text, but I think they probably _should_ have alt text.

Reply:
Yes, my thinking with making it non-optional is that we want it to error, so it's easier to spot where we're missing them.

Reply:
Kk. Sounds good.

Reply:
Accepted the changes.